RDF.js

```diff
@@ -48,105 +47,113 @@ function detectImport() {
 	if (nodes) {
 		return true;
 	}
+	return false;
 }
 
 var rdf = "http://www.w3.org/1999/02/22-rdf-syntax-ns#";
 
 var n = {
-	bib:"http://purl.org/net/biblio#",
-	bibo:"http://purl.org/ontology/bibo/",
-	dc1_0:"http://purl.org/dc/elements/1.0/",
-	dc:"http://purl.org/dc/elements/1.1/",
-	dcterms:"http://purl.org/dc/terms/",
-	prism:"http://prismstandard.org/namespaces/1.2/basic/",
-	prism2_0:"http://prismstandard.org/namespaces/basic/2.0/",
-	prism2_1:"http://prismstandard.org/namespaces/basic/2.1/",
-	foaf:"http://xmlns.com/foaf/0.1/",
-	vcard:"http://nwalsh.com/rdf/vCard#",
-	vcard2:"http://www.w3.org/2006/vcard/ns#",	// currently used only for NSF, but is probably
-							// very similar to the nwalsh vcard ontology in a
-							// different namespace
-	link:"http://purl.org/rss/1.0/modules/link/",
-	z:"http://www.zotero.org/namespaces/export#",
-	eprints:"http://purl.org/eprint/terms/",
-	og:"http://ogp.me/ns#",				// Used for Facebook's OpenGraph Protocol
-	article:"http://ogp.me/ns/article#",
-	book:"http://ogp.me/ns/book#",
-	so:"http://schema.org/",
-	codemeta:"https://codemeta.github.io/terms/"
+	bib: "http://purl.org/net/biblio#",
+	bibo: "http://purl.org/ontology/bibo/",
+	dc1_0: "http://purl.org/dc/elements/1.0/", // eslint-disable-line camelcase
+	dc: "http://purl.org/dc/elements/1.1/",
+	dcterms: "http://purl.org/dc/terms/",
+	prism: "http://prismstandard.org/namespaces/1.2/basic/",
+	prism2_0: "http://prismstandard.org/namespaces/basic/2.0/", // eslint-disable-line camelcase
+	prism2_1: "http://prismstandard.org/namespaces/basic/2.1/", // eslint-disable-line camelcase
```

Comment:
That is kind of cheating, because otherwise the linting will throw an error that we have here variable name with an underscore instead of using camelCase. @dstillman Do you have any alternative solution for naming the variables here? Or is this fine for you?

Reply:
No, that's exactly right.

https://github.com/zotero/eslint-config-zotero/issues/2